zabbix-java-gateway/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+logfile=/var/log/zabbix/zabbix_java_gateway.log
+logdir=$(dirname $logfile)
+pidfile=/var/run/zabbix/zabbix_java_gateway.pid
+piddir=$(dirname $pidfile)
+mkdir -p $piddir $logdir
+touch $logfile
+chown -R zabbix:zabbix $piddir $logfile
+/etc/init.d/zabbix-java-gateway start
+sleep 1
+tail -f $logfile --pid=$(cat $pidfile)
```

Comment:
@lins05 it worked for me in most cases, but your question is reasonable, what is a better value from your point of view? 3s/5s?